.pre-commit-config.yaml

```diff
@@ -30,3 +30,12 @@ repos:
           - --fix
       # Formatter
       - id: ruff-format
+  - repo: local
+    hooks:
+      - id: check-sort-dbt-yaml-files
+        name: check-sort-dbt-yaml-files
+        entry: python3 dbt/scripts/check_sort_dbt_yaml_files.py
+        language: system
+        types_or: [yaml, markdown]
+        files: ^dbt/
+        exclude: venv
```

Comment:
I double-checked the filenames generated by this config when `--all-files` is present by comparing it to the output of this `find` command and confirming they match:

```
find dbt/ \( -path "dbt/.venv" -o -path "dbt/node_modules" \) -prune -o -type f \( -name "*.yaml" -o -name "*.md" -o -name "*.yml" \)
```

I don't really understand how `node_modules` gets excluded from pre-commit, but it does, and the file paths match 🤷🏻‍♀️